plugins/fortinet_fortigate/help.md

```diff
@@ -735,6 +736,8 @@ Add the orchestrator's IP address to the trusted hosts in CIDR form e.g. `198.51
 
 # Links
 
+* [Fortinet FortiGate](https://www.fortinet.com/)
```

Comment:
Do we the same link here twice or would it be better to have it only in links?
I believe we need it in links or it fails the validator